packages/common/src/chains/index.ts

```diff
@@ -15,13 +16,15 @@ export function _getInitializedChains(customChains?: Chain[]) {
     '4': 'rinkeby',
     '42': 'kovan',
     '5': 'goerli',
+    '34180983699157880': 'yolov3',
```

Comment:
I am assuming that these numbers are supposed to be the chain id?

Reply:
Yes.